src/components/Header.module.scss

```diff
@@ -1,9 +1,11 @@
+@import "src/styles/color";
+
 .header {
   display: flex;
-  align-items: center;
-  padding: 20px;
-  height: 75px;
   width: 100%;
-  background-image: linear-gradient(to right, #33b3a6, #40e0d0);
+  height: 75px;
+  padding: 20px;
   font-size: 30px;
+  align-items: center;
```

Comment:
Just FYI: https://webdesign.tutsplus.com/outside-in-ordering-css-properties-by-importance--cms-21685a

no need to be dogmatic. But just keep that in mind

Reply:
updated